packages/services/api/src/modules/operations/providers/traces.ts

```diff
@@ -20,11 +22,12 @@ export class Traces {
     private clickHouse: ClickHouse,
     private logger: Logger,
     private storage: Storage,
+    @Inject(OTEL_TRACING_ENABLED) private otelTracingEnabled: boolean,
   ) {}
 
   async viewerCanAccessTraces(organizationId: string) {
     const organization = await this.storage.getOrganization({ organizationId });
-    return organization.featureFlags.otelTracing;
+    return organization.featureFlags.otelTracing || this.otelTracingEnabled;
```

Comment:
this may be out of scope, but should this also check the user permissions or is that being done elsewhere already?

Reply:
Yes, user permissions are already being checked elsewhere.

From what I've just analyzed, `viewerCanAccesTraces` is a field on the `Target` type. To access a `Target`, one must go through `target` query which calls `getTargetByReferenceInput`, which calls `getTarget`, which performs permission checking via `session.assertPerformAction` with the action `project:describe`. So basically by the time `viewerCanAccessTraces` is called, the user has already been permission checked to access target/project.